CONTRIBUTING.md

```diff
@@ -59,6 +63,10 @@ Here are some tips for getting your changes merged into xmonad:
     repository.  Include a new configuration file that shows off your
     changes if possible by creating a PR on that repository as well.
 
+  * Make sure you run the automated tests.  Both [xmonad-contrib][]
```

Comment:
I guess you are missing the link ?

Reply:
I thought the same, but everything is referenced this way (at the end of the file, all links are defined)

Reply:
TIL, thanks!